tests/x11/apache_ssl.pm

```diff
@@ -0,0 +1,76 @@
+# SUSE's Apache+SSL tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Enable SSL module on Apache2 server
+# Maintainer: Qingming Su <qingming.su@suse.com>
+
+use base "x11test";
+use testapi;
+use utils;
+use strict;
+
+sub quit_firefox() {
+    send_key "alt-f4";
+    if (check_screen("firefox-save-and-quit", 10)) {
+        send_key "ret";
+    }
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). This is a copy-paste from another module, but see comment below first before trying to change that.